src/blueapi/cli/cli.py

```diff
@@ -163,19 +166,16 @@ def get_devices(obj: dict) -> None:
 def listen_to_events(obj: dict) -> None:
     """Listen to events output by blueapi"""
     config: ApplicationConfig = obj["config"]
-    if config.stomp is not None:
-        event_bus_client = EventBusClient(
-            StompClient.for_broker(
-                broker=Broker(
-                    host=config.stomp.host,
-                    port=config.stomp.port,
-                    auth=config.stomp.auth,
-                )
+    assert config.stomp is not None, "By this point, the stomp config should be set"
```

Comment:
```suggestion
    assert config.stomp is not None, "Message bus needs to be configured"
```

nit: keep the exception message the same